### PR TITLE
Add git commit SHA to function labels

### PR DESCRIPTION
**Pull request by viveksyngh**
This changes adds a new label Git-SHA to functions deployed by openfaas
cloud.

Signed-off-by: Vivek Singh <vivekkmr45@yahoo.in>

Tested on local on kubernetes.

### CODE REVIEW

File: buildshiprun/handler.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ func Handle(req []byte) string {
 				"Git-Owner":      owner,
 				"Git-Repo":       repo,
 				"Git-DeployTime": strconv.FormatInt(time.Now().Unix(), 10), //Unix Epoch string
+				"Git-SHA":        getGitSHA(imageName),
```
Comment (alexellis) [inline buildshiprun/handler.go]: I was hoping we would pass it in from the previous method.. are we able to do that?

Comment (viveksyngh) [reply buildshiprun/handler.go]: oh yeah .. we can do that too. I think that would be a better way to do it.

**Comment by alexellis** [comment]
LGTM

**Comment by alexellis** [comment]
Tested - works fine

**Pull request merged**